types/pulltorefreshjs/index.d.ts

```diff
@@ -108,6 +113,20 @@ export interface Options {
      * Defaults to `t => Math.min(1, t / 2.5)`
      */
     resistanceFunction?(input: number): number;
+
+    /**
+     * Which condition should be met for pullToRefresh to trigger?
+     *
+     * Defaults to `!window.scrollY`
+     */
+    shouldPullToRefresh?(): boolean;
+
+    /**
+     * This value will be passed as `{ passive: true|false }` to `touchmove`
+     * listeners if passive-handlers are supported.
+     * (default: `false`)
```

Comment:
Can you keep this as "Defaults to `false`" to be consistent with the other comments?

Reply:
I changed this to the requested format, but there are other examples in this file for previous version too.